clause/limit.go

```diff
@@ -4,7 +4,7 @@ import "strconv"
 
 // Limit limit clause
 type Limit struct {
-	Limit  int
```

Comment:
This is a breaking change.

Reply:
then why is this released? I don't see the point where this change is necessary for the fix...